package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "@esy-ocaml/reason": ">=3.4.0",
-    "@opam/dune": "^1.9.0",
+    "@opam/dune": "^1.7.3",
```

Comment:
@bryphe @wokalski 

Just wondering if this should maybe be `>=1.7.3` since we don't mind using a higher version of dun in other scenarios where `brisk-reconciler` is getting used.